_drafts/shallow-clone-post-mortem.md

```diff
@@ -0,0 +1,56 @@
+[Recently](https://github.com/CocoaPods/CocoaPods/issues/4989) we were made aware of issues that our users were having
+cloning the [CocoaPods OSS spec-repo](https://github.com/CocoaPods/Specs).
```

Comment:
Yeah, it probably _should_, I guess I was trying to explicitly call for empathy by the reader here, instead of them thinking that we were using these resources purely for our own gains. I’m still on the fence on that.